utilities/backupable/backupable_db.cc

```diff
@@ -1710,7 +1725,8 @@ Status BackupEngineImpl::AddBackupFileWorkItem(
           "overwrite the file.",
           fname.c_str());
       need_to_copy = true;
-      backup_env_->DeleteFile(final_dest_path);
+      //**TODO: What to do on error?
+      backup_env_->DeleteFile(final_dest_path).PermitUncheckedError();
```

Comment:
I am not certain... but what is the downside of returning the error here?

Reply:
I do not know.  I did not return an error, as I did not want to change the underlying functionality for fear of what might break.